ui/session/session.go

```diff
@@ -58,6 +58,11 @@ func (s *Session) SetTheme(theme string) {
 	s.store.UpdateAppSessionField(s.sessionID, "theme", theme)
 }
 
+// SetCustomCSS updates the theme field in session.
+func (s *Session) SetCustomCSS(css string) {
+	s.store.UpdateAppSessionField(s.sessionID, "custom_css", css)
```

Comment:
Why that needs to be stored in the session?

Reply:
Sorry it doesnt, removed